test/e2e/multi-disk_test.go

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
also please rename this file to

`multi_disk_test.go`

Reply:
I totally meant to rename him.  Sorry about that.